hack/ci/install-binaries.sh

```diff
@@ -27,6 +27,8 @@ readonly GH_VERSION=${GH_VERSION:-2.0.0}
 readonly GH_CHECKSUM=${GH_CHECKSUM:-20c2d1b1915a0ff154df453576d9e97aab709ad4b236ce8313435b8b96d31e5c}
 readonly KUBECTL_TREE_CHECKSUM=${KUBECTL_TREE_CHECKSUM:-f4a3230d46b824889fca56525d051aad70815965a94623388ec0b5dc71781790}
 readonly KUBECTL_TREE_VERSION=${KUBECTL_TREE_VERSION:-0.4.1}
+readonly GRYPE_CHECKSUM=a0aaae28792a70fd465301cef0f3dc4bd09c2e707208f7a576e4085c8ea861d4
+readonly GRYPE_VERSION=0.27.2
```

Comment:
note that we _don't_ have to update grype on every grype database update - those are separate things (when you run it, it'll check if it needs to update of not: if so, then it does at runtime).